From cf5e5999b5f3ba7f860de845a19e7f48879e6a1d Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Fri, 7 Aug 2009 10:23:47 +0200
Subject: [PATCH 02/33] updated HP suspend quirks with s2ram whitelist entries from -r1.135

Updated HP suspend quirks with s2ram whitelist entries from -r1.135.
Added new:
- HP xw4550 Workstation
- HP xw6600 Workstation
- HP xw8600 Workstation
- HP xw9400 Workstation

tested by HP https://bugzilla.novell.com/show_bug.cgi?id=436284
---
 .../10freedesktop/20-video-quirk-pm-hp.fdi         |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fdi/information/10freedesktop/20-video-quirk-pm-hp.fdi b/fdi/information/10freedesktop/20-video-quirk-pm-hp.fdi
index 40fc620..37a4a99 100644
--- a/fdi/information/10freedesktop/20-video-quirk-pm-hp.fdi
+++ b/fdi/information/10freedesktop/20-video-quirk-pm-hp.fdi
@@ -151,7 +151,7 @@
           <merge key="power_management.quirk.vbemode_restore" type="bool">true</merge>
         </match>
       </match>
-      <match key="system.hardware.product" contains_outof="HP Compaq dc5800 Small Form Factor;HP d330 uT; nw9440;nw8440">
+      <match key="system.hardware.product" contains_outof="HP Compaq dc5800 Small Form Factor;HP d330 uT; nw9440;nw8440;HP xw4550 Workstation;HP xw6600 Workstation;HP xw8600 Workstation;HP xw9400 Workstation">
         <merge key="power_management.quirk.none" type="bool">true</merge>
       </match>
 
-- 
1.7.1

